docs/src/network.md

```diff
@@ -723,7 +723,9 @@ Important notes:
 
 ## WebSockets
 
-Playwright supports [WebSockets](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API) inspection out of the box. Every time a WebSocket is created, the [`event: Page.webSocket`] event is fired. This event contains the [WebSocket] instance for further web socket frames inspection:
+Playwright supports [WebSockets](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API) inspection, mocking and interception out of the box. See our [API mocking guide](./mock.md#mock-websockets) to learn how to mock WebSockets.
```

Comment:
nit: I'd like to add the `modify` world in there, seems easier to understand compared to intercept.